plugin/trino-pulsar/src/test/java/io/trino/plugin/pulsar/TestBasePulsarConnectorTest.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.pulsar;
+
+import com.google.common.collect.ImmutableMap;
+import io.trino.testing.BaseConnectorTest;
+import io.trino.testing.QueryRunner;
+import io.trino.testing.TestingConnectorBehavior;
+import org.testng.annotations.AfterClass;
+
+import static io.trino.plugin.pulsar.PulsarServer.SELECT_FROM_CUSTOMER;
+import static io.trino.plugin.pulsar.PulsarServer.SELECT_FROM_LINEITEM;
+import static io.trino.plugin.pulsar.PulsarServer.SELECT_FROM_NATION;
+import static io.trino.plugin.pulsar.PulsarServer.SELECT_FROM_ORDERS;
+import static io.trino.plugin.pulsar.PulsarServer.SELECT_FROM_REGION;
+
+public class TestBasePulsarConnectorTest
+        extends BaseConnectorTest
+{
+    protected PulsarServer pulsarServer;
+
+    @AfterClass(alwaysRun = true)
+    public void destroy()
+    {
+        if (pulsarServer != null) {
+            pulsarServer.close();
+        }
+    }
+
+    @Override
+    protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
+    {
+        return false;
+    }
+
+    @Override
+    public void testCreateTable()
```

Comment:
This makes an impression that test is passing while it is not not. If feature is not supported it should fail nicely. For example with message that `Feature is not supported in Pulsar connector` and this should be tested here. Usually we use `hasBehavior` to define what to expect from the connector and `BaseConnectorTest` should have proper assertions for postive and negative tests. If you are missing `TestingConnectorBehavior` for given test please add it and improve the test if needed.

Reply:
updated test so it now test the behavior properly.